src/main/java/org/broadinstitute/hellbender/tools/walkers/genotyper/MinimalGenotypingEngine.java

```diff
@@ -10,15 +10,15 @@
  * used only by the HaplotypeCaller for its isActive() determination. Should not be used for
  * any other purpose!
  */
-public final class MinimalGenotypingEngine extends GenotypingEngine<UnifiedArgumentCollection> {
+public final class MinimalGenotypingEngine extends GenotypingEngine<StandardCallerArgumentCollection> {
 
     /**
      * Creates a new unified genotyping given the UG configuration parameters and the targeted set of samples
```

Comment:
all these comments talk about the UG configuration which is confusing and outdated

Reply:
fixed